aws/templates/docker_compose_xchain.tftpl

```diff
@@ -15,8 +14,6 @@ services:
             entrypoint:
             - /bin/indexer
             - index
-            - --config-file=/config/${xchain_config_file_name}.json
-            - --omni-rpc=${omni_rpc}
```

Comment:
is this flag currently used in staging?

Reply:
No.

Reply:
yes, with latest deployed tag 0.1.2

Reply:
is it overwriting the value in the config file? Or just ignored

Reply:
Ignored